packages/layout/src/components/panel/es-sized-panel/es-sized-panel.tsx

```diff
@@ -0,0 +1,101 @@
+import { Component, h, Host, Watch, Element, Prop } from '@stencil/core';
+
+import {
+    bannerHeight,
+    cookieHeight,
+    panelHeight,
+    sidebarWidth,
+    toolbarWidth,
+} from '../../../utils/LayoutVar';
+import type { TargetableArea, TargetableEdge } from '../types';
+
+/**
+ * A panel that takes the size of it's content.
+ * Automatically sets the relevent layout var based on it's size.
```

Comment:
```suggestion
 * Automatically sets the relevant layout var based on it's size.
```

Reply:
I wonder if someone has made a spell checker for comments.

Reply:
They have, I'll give one a try.